pages/common/gh-gist.md

```diff
@@ -3,22 +3,22 @@
 > Work with GitHub Gists on the command-line.
 > More information: <https://cli.github.com/manual/gh_gist>.
 
-- Create a new Gist from a space-separated list of files:
+- Create a new Gist from specific files:
 
-`gh gist create {{path/to/files}}`
+`gh gist create {{path/to/file1 path/to/file2 ...(1..)}}`
 
-- Create a new Gist with a description:
+- Create a new Gist with a specific [desc]ription:
 
-`gh gist create {{path/to/file}} --desc "{{description}}"`
+`gh gist create {{path/to/file1 path/to/file2 ...(1..)}} --desc "{{description}}"`
 
-- Edit a Gist:
+- Edit a specific Gist:
 
-`gh gist edit {{id_or_url}}`
+`gh gist edit {{id|url}}`
 
 - List Gists owned by the currently logged in user:
 
-`gh gist list --limit {{int}}`
+`gh gist list --limit {{1..infinity}}`
```

Comment:
Not sure I see why this needs changing. `int` seems logical to me.

Reply:
It's standardization. :) I will replace it with `{{1..n}}`.

Reply:
Done.